File: backend/src/AtmosAQ.Application/Shared/MeasurementDto.cs
using System;

namespace AtmosAQ.Application.Shared
{
    public class MeasurementDto
    {
        public int LocationId { get; set; }
        
        public string Location { get; set; }
        
        public string Parameter { get; set; }
        
        public int Value { get; set; }
        
        public DateTime LastUpdated { get; set; } 
        
        public string Unit { get; set; }
        
        public string Country { get; set; }
        
        public string City { get; set; }
        
        public bool IsMobile { get; set; }
        
        public bool IsAnalysis { get; set; }
        
        public string Entity { get; set; }
        
        public string SensorType { get; set; }
    }
}

File: frontend/src/app/@core/models/measurement-dto.ts
import { CoordinateDto } from "./coordinate-dto";
import { DateDto } from "./date-dto";

export interface MeasurementDto {
    locationId?: number;
    location?: string | undefined;
    parameter?: string | undefined;
    value?: number;
    date?: DateDto;
    lastUpdated?: Date;
    unit?: string | undefined;
    coordinates?: CoordinateDto;
    country?: string | undefined;
    city?: string | undefined;
    isMobile?: boolean;
    isAnalysis?: boolean;
    entity?: string | undefined;
    sensorType?: string | undefined;
}